zerver/management/commands/sync_ldap_user_data.py

```diff
@@ -18,7 +18,7 @@
 # Run this on a cron job to pick up on name changes.
 @transaction.atomic
 def sync_ldap_user_data(
-    user_profiles: List[UserProfile], deactivation_protection: bool = True
+    user_profiles: Iterable[UserProfile], deactivation_protection: bool = True
```

Comment:
This is iterated on both line 25 and line 27, so should be `Collection[UserProfile]`.